payments/invoice/redirect.md

```diff
@@ -109,6 +109,7 @@ Content-Type: application/json
 |                  | └─➔&nbsp;`hostUrl`                | `array`       | The array of URLs valid for embedding of Swedbank Pay Hosted Views. If not supplied, view-operation will not be available.                                                                                                                                                                                            |
 | {% icon check %} | └─➔&nbsp;`completeUrl`            | `string`      | The URL that Swedbank Pay will redirect back to when the payer has completed his or her interactions with the payment. This does not indicate a successful payment, only that it has reached a final (complete) state. A `GET` request needs to be performed on the payment to inspect it further.                    |
 |                  | └─➔&nbsp;`cancelUrl`              | `string`      | The URI to redirect the payer to if the payment is canceled. Only used in redirect scenarios. Can not be used simultaneously with `paymentUrl`; only `cancelUrl` or `paymentUrl` can be used, not both.                                                                                                               |
+|                  | └─➔&nbsp;`paymentUrl`             | `string`      | This Parameter is used when the consumer is redirected out of the Seamless View (the iframe) and sent back after completing the payment. It should point to the page of where the Payment Order Seamless View is hosted. Read more about paymentUrl [here](https://developer.swedbankpay.com/checkout/other-features#payment-url-1).                                                          |
```

Comment:
Could we use "payer" instead of "consumer", so it corresponds with the rest of the table?

The link format we use elsewhere is [here][here]

[here]: /checkout/other-features#payment-url-1

Perhaps this instead?

"Read more about [paymentUrl][paymentUrl]"

[paymentUrl]: /checkout/other-features#payment-url-1

Reply:
The link format actually turned into links... 🤣 

Read more about [ paymentUrl ][ paymentUrl ]

[ paymentUrl ]: /checkout/other-features#payment-url-1

Reply:
It actually turned into links 🤣 

Read more about (paymentUrl)(paymentUrl)

(paymentUrl): /checkout/other-features#payment-url-1

Reply:
I think I got it, and made some changes based on the comments ✔️ 😺